Dockerfile

```diff
@@ -19,7 +26,9 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update -y --no-install-recommends &&
   libxrender1 \
   wget \
   ca-certificates \
-  curl && \
+  curl \
+  mandoc # For AWS CLI help \
+  unzip && \
```

Comment:
unzip is needed for the aws CLI, but it seems generally useful and isn't a big dependency.